static/external/js/Makefile

```diff
@@ -24,8 +24,8 @@ CDNJS = https://cdnjs.cloudflare.com/ajax/libs
 # Files #######################################################################
 
 all: angular.min.js angular.min.js.map angular-mocks.js angular-route.min.js angular-route.min.js.map \
-	 angular-ui-router.min.js django-angular.min.js icheck.min.js jquery.min.js jshint.js \
```

Comment:
@fferrin looks like jshint was accidentally removed? This may be causing issues.